BasicSteps/ProcessStep.cs

```diff
@@ -73,9 +78,27 @@ public Int32 Timeout
         ManualResetEvent outputWaitHandle, errorWaitHandle;
         StringBuilder output;
 
+        public ProcessStep()
+        {
+            Rules.Add(HasNoDuplicateEnvironmentVariables, "Cannot have multiple environment variables with the same name.", nameof(EnvironmentVariables));
```

Comment:
What do you think about "Environment variable names must be unique."?

Reply:
Yeah, that makes more sense. Fixed it